src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="app" :class="network">
+  <div id="app" :class="theme">
```

Comment:
@jbibla can we just do this?:

```suggestion
  <div id="app" class="lunie-dark" :class="network">
```

Reply:
I just tried it and it only picks `lunie-dark`